test/snapshots/array.test.js.md

```diff
@@ -123,34 +123,34 @@ Generated by [AVA](https://avajs.dev).
-        '<link rel="apple-touch-startup-image" media="(device-width: 375px) and (device-height: 667px) and (-webkit-device-pixel-ratio: 2) and (orientation: portrait)"    href="/apple-touch-startup-image-750x1334.png">',
-        '<link rel="apple-touch-startup-image" media="(device-width: 414px) and (device-height: 896px) and (-webkit-device-pixel-ratio: 2) and (orientation: portrait)"    href="/apple-touch-startup-image-828x1792.png">',
-        '<link rel="apple-touch-startup-image" media="(device-width: 375px) and (device-height: 812px) and (-webkit-device-pixel-ratio: 3) and (orientation: portrait)"    href="/apple-touch-startup-image-1125x2436.png">',
-        '<link rel="apple-touch-startup-image" media="(device-width: 414px) and (device-height: 736px) and (-webkit-device-pixel-ratio: 3) and (orientation: portrait)"    href="/apple-touch-startup-image-1242x2208.png">',
-        '<link rel="apple-touch-startup-image" media="(device-width: 414px) and (device-height: 896px) and (-webkit-device-pixel-ratio: 3) and (orientation: portrait)"    href="/apple-touch-startup-image-1242x2688.png">',
-        '<link rel="apple-touch-startup-image" media="(device-width: 768px) and (device-height: 1024px) and (-webkit-device-pixel-ratio: 2) and (orientation: portrait)"   href="/apple-touch-startup-image-1536x2048.png">',
+        '<link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png"/>',
+        '<link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png"/>',
+        '<link rel="icon" type="image/png" sizes="48x48" href="/favicon-48x48.png"/>',
+        '<link rel="manifest" href="/manifest.json"/>',
+        '<meta name="mobile-web-app-capable" content="yes"/>',
+        '<meta name="theme-color" content="#fff"/>',
+        '<meta name="application-name"/>',
+        '<link rel="apple-touch-icon" sizes="57x57" href="/apple-touch-icon-57x57.png"/>',
+        '<link rel="apple-touch-icon" sizes="60x60" href="/apple-touch-icon-60x60.png"/>',
+        '<link rel="apple-touch-icon" sizes="72x72" href="/apple-touch-icon-72x72.png"/>',
+        '<link rel="apple-touch-icon" sizes="76x76" href="/apple-touch-icon-76x76.png"/>',
+        '<link rel="apple-touch-icon" sizes="114x114" href="/apple-touch-icon-114x114.png"/>',
+        '<link rel="apple-touch-icon" sizes="120x120" href="/apple-touch-icon-120x120.png"/>',
+        '<link rel="apple-touch-icon" sizes="144x144" href="/apple-touch-icon-144x144.png"/>',
+        '<link rel="apple-touch-icon" sizes="152x152" href="/apple-touch-icon-152x152.png"/>',
+        '<link rel="apple-touch-icon" sizes="167x167" href="/apple-touch-icon-167x167.png"/>',
+        '<link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon-180x180.png"/>',
+        '<link rel="apple-touch-icon" sizes="1024x1024" href="/apple-touch-icon-1024x1024.png"/>',
+        '<meta name="apple-mobile-web-app-capable" content="yes"/>',
```

Comment:
I think we should avoid `/` here and introduce the `html5` option, but we can do it in separate PR

Reply:
Although they are considered useless as per the HTML5 standard, they still carry the meaning of being closed immediately with no content

Reply:
Let's avoid them anyway, I'm sure users will create a problem about this.